### PR TITLE
Covalence Repair Changes

### PR DESCRIPTION
**Pull request by Lilylicious**
Instead of a "Correct recipe repairs full", it consumes EMC and repairs the equivalent damage.

### CODE REVIEW

File: src/main/java/moze_intel/projecte/utils/EMCHelper.java [hunk 1]
```diff
@@ -261,4 +261,21 @@ private static double getStoredEMCBonus(ItemStack stack) {
 		}
 		return 0;
 	}
+
+	public static int getEMCPerDurability(ItemStack stack){
+
+		int EMC;
```
Comment (williewillus) [inline src/main/java/moze_intel/projecte/utils/EMCHelper.java]: `emc` instead of `EMC`

**Comment by williewillus** [comment]
This diverges from EE2 behaviour but I think it's a good change to make

**Comment by Lilylicious** [comment]
It does diverge, but it plugs up a pretty major exploit. It's a change that makes sense too, it makes it so that you can't get more EMC out of a repair than you put in. Also makes it easier to use, since repair recipes now don't care how much dust or what type you put in, as long as the EMC put in exceeds the minimum required to repair at all.

File: src/main/java/moze_intel/projecte/utils/EMCHelper.java (hunk 1 above)
Comment (Lilylicious) [inline src/main/java/moze_intel/projecte/utils/EMCHelper.java]: Will include that in the final cleanup commit

**Comment by williewillus** [comment]
@Lilylicious might want to remake this since the 1.10 branch got deleted (autocloses anything targeting it)

**Comment by sinkillerj** [comment]
@williewillus Yea we already discussed it on IRC, Lily is gonna remake it after a few cleanup tweaks.

**Pull request closed without merging**